src/components/Header/index.tsx

```diff
@@ -2,7 +2,7 @@ import React, { FC } from 'react'
 import { Logo } from '../Logo/Logo'
 import styled from 'styled-components'
 import logo from '../Logo/images/vamalaka.svg'
-import searchIcon from '../../assests/searchIcon.svg'
+import searchIcon from '../../assets/searchIcon.svg'
```

Comment:
I am not sure about this but is it fine to use camel case with svg file like this or we can name it as we name our component? I mean, should it be `searchIcon` or `SearchIcon`?